### DISCUSSION TITLE
Gltf model problems (pearlescent)

### DISCUSSION THREAD

**Discussion by dsmaxville** [google/model-viewer#2198] [Q&A]
### Description

Im trying to use pearlescent effect on 3d model and on model-viewer its not working
[mac-10_2020_proba_09.zip](https://github.com/google/model-viewer/files/4508087/mac-10_2020_proba_09.zip)

#### Live Demo
https://csgohub.ru/3d-test/skins/Case_CS20/GOLO/mac-10_2020_proba_09/mac-10_2020_proba_09.html  - its working on verge3d web

https://csgohub.ru/3d/index_old.html - but on model-viewer is looking bad

https://csgohub.ru/3d/skins/mac10/mac-10_2020_proba_09.gltf  -gltf file

### Versions

- model-viewer: v0.10.0
- three.js: rXX

**Comment by donmccurdy** [comment]
The verge3d link appears broken here. Opening the model on a couple online viewers, it is consistent (other than lighting differences).

| three.js | babylonjs |
|---|---|
| ![Screen Shot 2020-04-21 at 10 26 02 PM](https://user-images.githubusercontent.com/1848368/79943958-310cda80-841f-11ea-960a-be5cfc14b37c.png) | ![Screen Shot 2020-04-21 at 10 26 15 PM](https://user-images.githubusercontent.com/1848368/79943960-323e0780-841f-11ea-85b9-755c1db346e7.png) |

The model contains many custom Verge 3D extensions:

    S8S_v3d_texture_data
    S8S_v3d_material_data
    S8S_v3d_camera_data
    S8S_v3d_data
    S8S_v3d_node_data
    S8S_v3d_scene_data

These are not part of the official glTF specification. No tools outside of those Verge3D provides are going to be able to understand them, and it's likely that these contain the material effects you are looking for. You will need to export a standard glTF file to use it in tools like model-viewer.

**Comment by dsmaxville** [comment]
Thank you for your reply!

Mb you know, Is it possible to keep the pearlescent effect in a standard glb (or gltf) file?

**PS link on Verge3D fixed**  [LINK](https://csgohub.ru/3d-test/skins/Case_CS20/GOLO/mac-10_2020_proba_09/mac-10_2020_proba_09.html)

**Comment by donmccurdy** [comment]
Not currently possible in standard glTF, sorry. I'm not sure but I believe the effect is the same as thin film iridescence, which is [being discussed](https://github.com/KhronosGroup/glTF/pull/1742) as a future addition.
